src/app/shared/conditional-show/conditional-show.directive.spec.ts

```diff
@@ -66,6 +67,20 @@ describe('ConditionalShowDirective', () => {
         expect(conditionalShow.condition).toBeUndefined();
     });
 
```

Comment:
the story mentions that the grey bar has to be displayed only if the show is triggered from field on the current page. 
If the field is displayed from fields of another page, grey bar should not be displayed. Can we have a test for this?